opencog/nlp/lg-dict/LGDictReader.cc

```diff
@@ -69,7 +71,36 @@ LGDictReader::~LGDictReader()
  * @param word   the input word string
  * @return       the handle to the newly created atom
  */
-Handle LGDictReader::getAtom(const std::string& word)
+Handle LGDictReader::getAtom(const std::string word)
```

Comment:
why are you removing the reference? Using the reference avoids a string copy, right?